### ISSUE TITLE
noticed woman names with male avatars

### ISSUE THREAD

**Issue by shimondoodkin** [Marak/faker.js#470]

**Comment by Marak** [comment]
Yeah, there is no way to distinguish between the UIFaces sources that I know of. cc @calebogden 

Duplicate: https://github.com/Marak/faker.js/issues/422

Will accept a pull request which sorts the data by gender.

**Comment by Marak** [comment]
I think someone would have to go through https://github.com/Marak/faker.js/blob/master/lib/locales/en/internet/avatar_uri.js and split them into separate arrays by gender.

**Comment by calebogden** [comment]
Yeah there's no way to get gender.

I'm looking into https://www.kairos.com as a solution but I don't have a time estimate on this

**Comment by Marak** [comment]
@calebogden -

I would recommend using https://www.mturk.com/mturk/welcome. You could probably get decent results if you paid a few cents per image and ran them through twice for accuracy. You'd also want to introduce a `neutral` gender for avatars that aren't gender specific.

Let me know if you have any questions or could use help getting this going.

**Comment by tomByrer** [comment]
Sorted, even has Lego
https://randomuser.me/photos

**Comment by Nisthar** [comment]
Its been 3 years
any updates?

**Comment by calebogden** [comment]
I'm not really supporting this project anymore other than hosting the images
